tests/plugins/conditional_formatting.test.ts

```diff
@@ -437,16 +437,14 @@ describe("conditional format", () => {
     });
   });
 
-  test("Set conditionalFormat on empty cell", () => {
+  test("Prevents creation of CF CellIsRule with operator 'Equal' and empty value", () => {
```

Comment:
Also add a test for `notEqual` . You could maybe add them both in the test `%s operator with missing first argument %s` where we seem to be testing all the others:)